beacon-chain/rpc/beacon_server.go

```diff
@@ -136,6 +148,109 @@ func (bs *BeaconServer) BlockTree(ctx context.Context, _ *ptypes.Empty) (*pb.Blo
 	}, nil
 }
 
+// Eth1Data is a mechanism used by block proposers vote on a recent Ethereum 1.0 block hash and an
+// associated deposit root found in the Ethereum 1.0 deposit contract. When consensus is formed,
+// state.latest_eth1_data is updated, and validator deposits up to this root can be processed.
+// The deposit root can be calculated by calling the get_deposit_root() function of
+// the deposit contract using the post-state of the block hash.
+func (bs *BeaconServer) Eth1Data(ctx context.Context, _ *ptypes.Empty) (*pbp2p.Eth1Data, error) {
```

Comment:
I dont think this needs to be exported?

```suggestion
func (bs *BeaconServer) eth1Data(ctx context.Context, _ *ptypes.Empty) (*pbp2p.Eth1Data, error) {
```

Reply:
+1 , eth1data is no longer a rpc method. We can remove the `_ *ptypes.Empty` too

Reply:
stopped exposing. removed the empty field